src/navigation-instruction.js

```diff
@@ -233,16 +233,20 @@ export class NavigationInstruction {
   }
 
   _buildTitle(separator: string = ' | '): string {
-    let title = this.config.navModel.title || '';
+    let title = '';
     let childTitles = [];
 
+    if (this.config.navModel.title) {
+      title = this.router.transformTitle(this.config.navModel.title);
+    }
+
     for (let viewPortName in this.viewPortInstructions) {
       let viewPortInstruction = this.viewPortInstructions[viewPortName];
 
       if (viewPortInstruction.childNavigationInstruction) {
         let childTitle = viewPortInstruction.childNavigationInstruction._buildTitle(separator);
         if (childTitle) {
-          childTitles.push(childTitle);
+          childTitles.push(this.router.transformTitle(childTitle));
```

Comment:
It seems like this would cause a double translation of the child title wouldn't it? The first would happen inside the _buildTitle, since it calls transformTitle internally, and then the second would happen on line 249. Is that correct or am I missing something? @manuel-guilbault